products/sle/main.pm

```diff
@@ -458,6 +458,9 @@ sub load_inst_tests() {
             loadtest 'installation/partitioning_full_lvm';
         }
         if (get_var("FILESYSTEM")) {
+            if (get_var('PARTITIONING_WARNINGS')) {
+                loadtest 'installation/partitioning_warnings';
+            }
```

Comment:
because my test development machine is busy with two other changes I am in the process of checking. It sounds to me like you don't want to test for openSUSE. Please see https://progress.opensuse.org/projects/suseqa/wiki#quotupstream-firstquot-approach we should really try to keep openSUSE in sync